### DISCUSSION TITLE
ORDINALS - BASE STANDARD FEATURES

### DISCUSSION THREAD

**Discussion by WILDM4N** [ordinals/ord#771] [General]
Draft of base standard functionality, by Rodarmor. Aim is to get to a feature complete state relatively quickly, let's say within a year, and set things up so that future innovations can be done on top of the base standard.

**BASE STANDARD FEATURES**

* Prime ordinals for iron-clad provenance. You can go to /ordinal/ORDINAL or /SYMBOL and see a list of inscriptions that a prime ordinal has made
* Each inscription is a file
* Each file probably needs a MIME type (that's how web servers work, when you make an HTTP request, the response has a MIME type)
* SYMBOLS/CODES so people can publish works with easy to link-to names
* Some way of linking to content off chain. This doesn't have to be done at first, but it's going to eventually enable more expressive content at the cost of expressivity
* When you load an inscription with type text/html, it can make web requests, but the only requests it can make are for other inscriptions that have previously been published. < img src=/FOOBAR> for example would return the inscription with symbol FOOBAR. This keeps inscriptions immutable, but allows building up a collective body of work and commons. You could even publish javascript libraries for use in other inscriptions.
* one-of-one inscriptions should be possible
* duplicate inscriptions should be possible (same inscription on multiple ordinals)
* unique inscriptions should be possible (multiple inscriptions on multiple ordinals)
* open-ended collections should be possible (probably with prime ordinals, you can use sub-prime ordinal to represent collection, and issue works with it.)

**Comment by GaloisField2718** [comment]
Hello, 

Concerning MIME type attached to a file (I think which it referenced or linked with an inscription). 
How can use it to make inscriptions ? Inscribe this MIME ?

Is SYMBOLS/CODES still exists ?

For type/html /FOOBAR must refer to inscription number or url inscription or preview/nb_inscription ?

You told different type of inscriptions at the end. We are agree that no inscription could be overwritten ?
